app/components/projects-list/project/menu/component.js

```diff
@@ -3,12 +3,25 @@ const { Component, inject } = Ember;
 
 export default Component.extend({
   notify: inject.service(),
+  terminal: inject.service(),
+  ideName: 'code',
```

Comment:
Hmm, hardcoded :/ I will push my changes related with settings but is still need to be tested. If you would like to practice, go ahead.